man/sigrok_rdm_sniffer.1

```diff
@@ -0,0 +1,48 @@
+.TH sigrok_rdm_sniffer 1 "March 2021"
+.SH NAME
+sigrok_rdm_sniffer \- Decode DMX/RDM data from devices supported by sigrok
+.SH SYNOPSIS
+.B sigrok_rdm_sniffer
+[ options ]
+.SH DESCRIPTION
+.B sigrok_rdm_sniffer
+Decode DMX/RDM data from devices supported by sigrok
```

Comment:
same as line 3

Reply:
Yeah they all come from the same place via magic like this:
`./tools/logic/sigrok_rdm_sniffer --gen-manpage > man/sigrok_rdm_sniffer.1`

I've fixed the source and updated it all.